benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,272 @@
+title: "Comparing embedding layers in spaCy"
+description: |
+  This project compares `MultiHashEmbed` with its standard embedding counterpart
+  `MultiEmbed` for Named Entity Recognition. In order to access the datasets,
+  you first need to clone the [`spancat-datasets`](https://github.com/explosion/spancat-datasets)
+  repository:
+
+  ```sh
+  # Clone the spancat-datasets repo to access the datasets
+  git clone git@github.com:explosion/spancat-datasets.git
+  ```
+
+  and run the necessary conversion scripts. For example, let's perform the
+  conversion command for the [Anatomical Entity Mention (AnEM)](http://www.nactem.ac.uk/anatomy/)
+  corpus:
+
+  ```sh
+  # While inside the spancat-datasets repository
+  spacy project run anem
+  ```
+
+  This will generate the spaCy files that you can use for training. Once done,
+  you **should copy these files to this project**. For example, you can perform
+  a directory copy in Linux via:
+
+  ```sh
+  cp -r spancat-datasets/corpus/ner/*. ner_embeddings/corpus/. 
+  ```
+
+  You can now supply the local path to the commands and workflows to
+  perform experiments in **this project:**
+
+  ```sh
+  # Perform experiments in the ner_embeddings project
+  spacy project run train . --vars.dataset anem
+  ```
+
+vars:
+  ner_config: "ner_multiembed"
+  dataset: "anem"
+  language: "en"
+  vectors: "vectors/fasttext-en"
+  tables_path: "tables"
+  metrics_dir: "metrics"
+  attrs: null
+  rows: "[5000, 2500, 2500, 2500]"
+  include_static_vectors: true
+  min_freq: 10
+  gpu_id: 0
+  seed: 42
+  batch_size: 1000
+  num_hashes: 4
+
+directories:
+  - "assets"
+  - "configs"
+  - "corpus"
+  - "scripts"
+  - "training"
+  - "metrics"
+  - "vectors"
+  - "tables"
+  - "debug"
+
+workflows:
+  setup:
+    - "download-models"
+    - "init-fasttext"
+    - "make-tables"
+
+assets:
```

Comment:
Should we include that `spancat-datasets` github repo into the `assets` and then it gets cloned automatically and you don't have to clone by hand?

Reply:
That actually makes sense!

Reply:
Sudden thought, if we're going to use [this PR](https://github.com/explosion/projects/pull/154) instead of the [span-labeling-datasets](https://github.com/explosion/span-labeling-datasets) repo, then it may be easier because they don't need to clone anything (they belong to the same repo) 💯 

I can then update the `commands` to do all the processing and copying instead. What do you think?